kb/sd_of_ann_actions/concepts/concept_all_layer_neurons_activation_function_calculating_action.scs

```diff
@@ -0,0 +1,36 @@
+concept_all_layer_neurons_activation_function_calculating_action
```

Comment:
Rename actions prefixes from concept_ to action_. For this:
concept_all_layer_neurons_activation_function_calculating_action to action_calculate_all_layer_neurons_activation_function